### ISSUE TITLE
Code review

### ISSUE THREAD

**Issue by BigDeepBlue** [alena-kono/graduate_work#61]
Добрый день!

Отличный у вас диплом получился!!! Видно, что был проделан большой объем работы. У меня совсем немного рекомендаций.

1. [Тут](https://github.com/alena-kono/graduate_work/blob/main/cdn_api/cdn_api/utils/authorization.py#L24) ваш код перехватывает все исключения и возвращает None в случае любой ошибки. Это может затруднить отладку, так как неясно, какая именно ошибка произошла. Рекомендуется логировать исключения или обрабатывать их более специфически.
2. Тут я бы все таки рекомендовал не использовать конкретные зависимости (rabbit). Лучше что-то абстрактное в самой логике. Прокидывать раббит при инициализации.
3. [Папку](https://github.com/alena-kono/graduate_work/tree/main/cdn_api/tests) с тестами создали, но она пустая :)  Лучше, конечно, хотя бы базовые тесты написать.
4. [Можно лучше] Порекомендовал бы еще добавить логов по коду уровня info, debug для того чтобы понимать, что происходит при работе вашего приложения в продакшене. Представьте, что вы свой проект выводите в production, и реальные клиенты массово начинают этим пользоваться. При первом релизе проекта очень удобно когда он покрыт логами info и debug для понимания того, что происходит. Практически всегда бывает, что разработчик не все случаи оттестировал, и когда реальные клиенты начинают пользоваться сервисом, что-то обязательно всплывает.
5. Не нашел конфига docker-compose для production. Можете оставить в ридми комментарий по этому поводу, если я проглядел

**Comment by alena-kono** [comment]
Добрый день ! Спасибо за ревью!
Поправили/улучшили 1,3,4,5 пункты.

По поводу пункта 2 не совсем поняли, о каком коде речь. Просмотрели все, и не нашли ошибку.

**Comment by BigDeepBlue** [comment]
да, там ссылочка не прикрепилась по 2. Ну там "можно лучше".  https://github.com/alena-kono/graduate_work/blob/main/cdn_api/cdn_api/uow/video.py#L55

Принято, удачи на защите!
